ci-operator/step-registry/load-balancer/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- emilienm
```

Comment:
who else could be an approver?

Reply:
Leads of all on-prem platforms at least.